packages/lib-subject-viewers/src/VolumetricViewer/components/Plane.js

```diff
@@ -227,18 +232,31 @@ export const Plane = ({
 
   return (
     <StyledBox className={`plane-container plane-container-${dimension} ${expanded ? 'expanded' : 'collapsed'} no-select`} ref={containerRef}>
-      <Box className={`plane-title ${expanded ? 'expanded' : 'collapsed'}`}>
+      <button
+        aria-controls={`section-${planeId}`}
+        aria-expanded={expanded}
```

Comment:
```suggestion
        aria-expanded={expanded.toString()}
```

ARIA states are strings eg. `'true'|'false'`. I think that trips a lot of people up.